util.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+var fs = require('fs')
+var path = require('path')
+
+module.exports.findIncludePath = function (originPath, includePath) {
```

Comment:
Can we move this to a private static member of GoogleProtoFilesRoot?

Reply:
I wanted to only expose this method. Placing this as a private static member of GoogleProtoFilesRoot will require me to expose the class for testing (unless theres a way to only expose the class to tests that I don't know about 😄).

Reply:
I think of `load.js` as "private" in the sense that the user isn't accessing it directly. Currently, index.js exposes the "load" functionality, and it can continue to do that if we make the load.js file fully-functioning by moving `findIncludePath` onto it. I'm just going for as contained as possible, so things that are related can stay closely connected-- it seems like only load.js will ever need to use `findIncludePath`, so I don't think it needs to be factored out into another file.

Reply:
Understood! Working on the fix rn.

Reply:
Done!